ts/features/wallet/onboarding/bancomat/utils/abi.ts

```diff
@@ -0,0 +1,6 @@
+export const sortAbiByName = (abis: ReadonlyArray<any>) =>
+  abis
+    .concat()
```

Comment:
Are you sure use `concat`?

```typescript
export const sortAbiByName = (abis: ReadonlyArray<Abi>) =>
  [...abis].sort((abi1: Abi, abi2: Abi) => {
    const abi1Name = abi1?.name || "";
    const abi2Name = abi2?.name || "";
    return abi1Name
      .toLocaleLowerCase()
      .localeCompare(abi2Name.toLocaleLowerCase());
  });
```

Furthermore you can leave a comment describing this function?